src/styles/style.css

```diff
@@ -0,0 +1,163 @@
+.body {
+  margin: 0;
+
+  --border-color: rgb(240, 7, 228);
+  --hover-color: blue;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 199px;
+  height: 100%;
+  border: 1px solid var(--border-color);
```

Comment:
According to the Code Style Rules, there should be no space between property value and unit when the value is zero (e.g., `margin: 0;`).